samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -809,7 +823,8 @@ int link_shell(const struct shell *shell, size_t argc, char **argv)
 				shell_error(
 					shell,
 					"Not a valid number for --threshold_time (milliseconds).");
-				return -EINVAL;
+				ret = -EINVAL;
+				goto return_without_usage;
```

Comment:
Is this change clearer than the original code, or what's the intention here? I'd say it's easier to read if we just return here, or am I missing some functional change?

Reply:
Removed the change.